auction_keeper/main.py

```diff
@@ -80,7 +80,7 @@ def __init__(self, args: list, **kwargs):
         parser.add_argument('--deal-for', type=str, nargs="+",
                             help="List of addresses for which auctions will be dealt")
 
-        parser.add_argument('--min-auction', type=int, default=0,
+        parser.add_argument('--min-auction', type=int, default=1,
```

Comment:
For all auction types, the first auction is id=1; don't waste resources checking auction 0.